package.json

```diff
@@ -21,31 +21,31 @@
     "build:watch": "tsc -w -p tsconfig.release.json"
   },
   "devDependencies": {
-    "@types/common-tags": "^1.8.0",
-    "@types/jest": "^26.0.23",
-    "@types/node": "^16.11.22",
-    "@types/ws": "^7.4.4",
-    "husky": "^6.0.0",
-    "jest": "^27.0.4",
-    "plop": "^3.0.5",
+    "@types/common-tags": "^1.8.1",
+    "@types/jest": "^29.5.3",
+    "@types/node": "^20.4.6",
+    "@types/ws": "^8.5.5",
+    "husky": "^8.0.3",
+    "jest": "^29.6.2",
+    "plop": "^3.1.2",
     "prettier": "^2.3.0",
```

Comment:
`prettier` cannot be updated to latest v3 because of https://github.com/prettier/eslint-plugin-prettier/issues/562

Reply:
Can we only upgrade packages related to discord.js for now?

Reply:
Actually I tried this first but it threw me into dependency hell and I got a whole lot of version conflicts and compiler errors. I can take a look at it tomorrow again.

Reply:
Following necessary dependency upgrades: 
* `discord.js 14.12` needs `typescript 5.1.6`
* `ts-jest 27.0.3` needs `typescript < 5.0` => upgrade to `28.0.0`
* `ts-jest 28.0.0` needs `jest 28.0.0`